packages/wonder-blocks-birthday-picker/src/components/birthday-picker.js

```diff
@@ -72,6 +114,9 @@ export default class BirthdayPicker extends React.Component<Props, State> {
         let year = null;
         let error = null;
 
+        // merge custom labels with the default ones
+        this.labels = {...defaultLabels, ...props.labels};
```

Comment:
Since all of the fields in `Labels` are required this could be `this.labels = props.labels ?? defaultLabels`.